### ISSUE TITLE
Useless text displayed before user comment in OSD

### ISSUE THREAD

**Issue by rouelle** [BestImageViewer/geeqie#805]
### ISSUE TYPE

* Bug Report

### GEEQIE VERSION

```
Geeqie 1.5.1
```

### OS / DISTRIBUTION

Geeqie 1.5.1-3 from Arch Linux

### SUMMARY
Since Geeqie 1.5.1 the OSD window shows **_charset=Unicode_** before the comment text, a behavior that was never present before. Here is my OSD model in Geeqie's settings:

%name%
<b>%Exif.Photo.UserComment%</b>
%collection:<i>*</i>\n%(%number%/%total%)

I'm using this model since years but the behavior change is fairly recent...

### STEPS TO REPRODUCE

Simply show any picture with UserComment metadata.
![geeqie_151-3](https://user-images.githubusercontent.com/46628603/92648212-18f70400-f2b7-11ea-9fb8-6899715b1010.png)

**Comment by caclark** [comment]
It looks to me that the "charset=Unicode" string is coming from exiv2.
That is confusing, because I don't think that part of exiv2 has changed for years.

**Comment by rouelle** [comment]
exiv2 was upgraded to version exiv2-0.27.3-1 (Arch Linux 64) on 20-08-30 and that seems to correspond to the moment I noticed the new behavior of adding "charset=Unicode" string before the comment. So I downgrade to version 0.27.2 and the useless string disappeared!

![geeqie_151-3_with_exiv2downgrade](https://user-images.githubusercontent.com/46628603/92847845-986bfc80-f3b7-11ea-9395-eded5c69826b.png)

That's a good hint...

**Comment by caclark** [comment]
I now see that this is discussed in:
https://github.com/Exiv2/exiv2/issues/1258

On my system, I cannot get Geeqie to write UserComment. Do you use Geeqie or another program to write UserComment?

**Comment by rouelle** [comment]
I usually write my comments in a photo using Gimp. Gimp writes strings in File:Comment.
But in Geeqie I never succeed in reading FileComment to be showed in OSD, so I use this batch command to copy in UserComment my FileComment:

`exiftool "-EXIF:UserComment<FILE:Comment" *.jpg -overwrite_original`

Then my OSD model works with Geeqie. I also use Fotoxx to edit a typo or to correct a comment when I find one because it doesn't affect the quality of the image. I tried to modify my OSD model to read File:Comment instead but, like I said, it never succeed. I work this way since 10 years...

**Comment by caclark** [comment]
With reference to the Overlay Screen Display, if you bring up the Exif window (View/Exif Window) you may see if Gimp is using a different tag.
In a quick check I made, it seems that Gimp uses the tags:
`Exif.Image.ImageDescription`
`Exif.Photo.UserComment`

And the OSD also has the formatted tag:
`%imagecomment%`

**Comment by rouelle** [comment]
Ah, you are right! Using the tag %imagecomment% in Geeqie resolves my problem :-) Previously I tried with the tag %comment% and different variants involving comment with capital C, Exif, Image, Photo, etc. Well, the solution was simple.

I upgraded exiv to exiv2-0.27.3-1 and using %imagecomment% Geeqie doesn't show the "charset=Unicode" in OSD.

So my issue may be marked as solved. Thank You - Merci ;-)
